### DISCUSSION TITLE
https://thanhle.blog/vi/blog/great-power-come-with-greate-responsibility

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#16] [Ideas]
# https://thanhle.blog/vi/blog/great-power-come-with-greate-responsibility

https://thanhle.blog/vi/blog/great-power-come-with-greate-responsibility

**Comment by nguyenhoaikhang37** [comment]
thật sự rất thích viết blog của anh Thành, cách viết của anh cuốn thực sự (không lan man, thấm nhanh cũng như có sự hài hước trong đó), mong sau anh Thành sẽ ra 1 cuốn sách gì đó thì em sẽ ủng hộ đầu tiên kk

**Reply by thanhlmm** [reply]
Haha. Thanks em. Sai chính tả nhiều lắm sao viết sách dc e :))))

**Reply by nguyenhoaikhang37** [reply]
haha bug cũng là 1 tính năng mà anh, quan trọng là ng đọc tiếp cận đc nội dung mà a kk, em đọc blog anh kiểu bị cuốn vào văn viết anh ghê luôn, mong sẽ đọc đc thật nhiều bài blog ở trên website của anh hơn nữa (đặc beitej em quan tâm nhất về FE và react ạ hihi^^)

**Comment by 19522515** [comment]
Đọc cuốn hơn bánh tráng cuốn nữa! Great motivation....

**Reply by thanhlmm** [reply]
Thanks bạn. Nhớ subcribe email để nhận bài viết mới mỗi tuần nha 😇
